src/main/webapp/js/stats-page-loader.js

```diff
@@ -0,0 +1,24 @@
+// fetch stats and display them in page
```

Comment:
Not super comfortable with js, can we use `/** <comment> */` comment style here?

Reply:
Yep, done!